tests/stl/hash_check.cpp

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
Define the template classes you are using in this test on top so that it can be included in our coverage